ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -124,8 +125,11 @@ message CreatedEvent {
   // after the introduction of explicit disclosure.
   // Deprecated in favor of ``created_event_blob``.
   // Optional
-
   ContractMetadata metadata = 10 [deprecated = true];
+
+  // Ledger effective time of the transaction that created the contract.
+  // Required
+  google.protobuf.Timestamp created_at = 14;
```

Comment:
@tudor-da: just wondering why this can't just be incorporated into the blob, for explicit disclosure purposes?

Reply:
indeed. The data is in the blob, why do we need to duplicate it ?

Reply:
As the blob is opaque, for completeness of the exposed information to the client, right? e.g. CN uses it

Reply:
The ledger-effective time is part of a contract's state, as it's required to check causal monotonicity. Not having it on a created-event was an oversight.

Reply:
Thx @tudor-da @meiersi-da for the answers - this makes sense. It is orthogonal to explicit disclosure - and you want it to be visible for other purposes, so we should always collect/provide it.